<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NightMessage</class>
 <widget class="QDialog" name="NightMessage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>120</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Night Notes</string>
  </property>
  <property name="windowIcon">
   <iconset resource="NightNoteRes.qrc">
    <normaloff>:/images/icon256.png</normaloff>:/images/icon256.png</iconset>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>85</y>
     <width>401</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::NoButton</set>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>68</width>
     <height>68</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelTitle">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="margin">
    <number>3</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelText">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>33</y>
     <width>320</width>
     <height>51</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>320</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="margin">
    <number>3</number>
   </property>
   <property name="indent">
    <number>-1</number>
   </property>
   <property name="openExternalLinks">
    <bool>true</bool>
   </property>
  </widget>
  <action name="actionYes">
   <property name="text">
    <string>Yes</string>
   </property>
   <property name="toolTip">
    <string>Yes</string>
   </property>
   <property name="shortcut">
    <string>Y</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="NightNoteRes.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NightMessage</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NightMessage</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
